docs/vdc/networking-virtual-datacenter.md

```diff
@@ -10,360 +10,375 @@ ms.date: 09/24/2018
 
-Deploying ExpressRoute connections usually involves engaging with an ExpressRoute service provider. For customers that need to start quickly, it is common to initially use Site-to-Site VPN to establish connectivity between the VDC and on-premises resources, and then migrate to ExpressRoute connection when your physical interconnection with your service provider is complete.
+Deploying ExpressRoute connections usually involves engaging with an ExpressRoute service provider. For customers that need to start quickly, it's common to initially use a site-to-site VPN to establish connectivity between the VDC and on-premises resources. Then migrate to ExpressRoute connection when your physical interconnection with your service provider is finished.
 
-#### *Connectivity within the cloud*
-[VNets][VNet] and [VNet Peering][VNetPeering] are the basic networking connectivity services inside a VDC. A VNet guarantees a natural boundary of isolation for VDC resources, and VNet peering allows intercommunication between different VNets within the same Azure region or even across regions. Traffic control inside a VNet and between VNets need to match a set of security rules specified through Access Control Lists ([Network Security Group][NSG]), [Network Virtual Appliances][NVA], and custom routing tables ([UDR][UDR]).
+#### Connectivity within the cloud
+[Virtual networks][VNet] and [virtual network peering][VNetPeering] are the basic networking connectivity services inside a VDC. A virtual network guarantees a natural boundary of isolation for VDC resources. And virtual network peering allows intercommunication between different virtual networks in the same Azure region or even across regions. Traffic control inside a virtual network and between virtual networks needs to match a set of security rules specified through access control lists. See information on [network security groups][NSG], [network virtual appliances (NVAs)][NVA], and [custom routing tables for user-defined routes][UDR].
 
-## Virtual Data Center overview
+## Virtual datacenter overview
 
 ### Topology
-_Hub and spokes_ is a model for extending a Virtual Data Center within a single Azure region.
+**Hub and spokes** is a model for extending a virtual datacenter within a single Azure region.
 
 [![1]][1]
 
-The hub is the central zone that controls and inspects ingress and/or egress traffic between different zones: Internet, on-premises, and the spokes. The hub and spoke topology gives the IT department an effective way to enforce security policies in a central location, while reducing the potential for misconfiguration and exposure.
+The hub is the central zone that controls and inspects ingress or egress traffic between different zones: internet, on-premises, and the spokes. The hub and spoke topology gives the IT department an effective way to enforce security policies in a central location. It also reduces the potential for misconfiguration and exposure.
```

Comment:
"The hub is the central zone..." should be "A hub is the central network zone..."